### PR TITLE
updated images layout for tutorial section

### PR DESCRIPTION
**Pull request by n-orlowski**

### CODE REVIEW

File: content/en/apps/tutorials/contact-and-users-1.md [hunk 1]
```diff
@@ -44,91 +44,133 @@ In this tutorial, you will work with the default contact forms and the default h
...
+{{< figure src="new-chw-user/app-settings.png" link="new-chw-user/app-settings.png" class="right col-6 col-lg-8" >}}
+
 Go to the **hamburger menu** and select **App Settings**.
 
-![app settings](new-chw-user/app-settings.png "App settings") <br><br>
+<br clear="all">
+
+{{< figure src="new-chw-user/add-user.png" link="new-chw-user/add-user.png" class="right col-6 col-lg-8" >}}
 
 When you are on the **App Settings** page, select **Users** on the right hand side and then select **Add User**.
```
Comment (smbuthia) [inline content/en/apps/tutorials/contact-and-users-1.md:161]:
```suggestion
When you are on the **App Settings** page, select **Users** on the left-hand side and then select **Add User** on the right-hand side.
```

**Review by smbuthia — CHANGES_REQUESTED** [review]
Great improvements! Thanks. Just one small correction before merging.

**Comment by n-orlowski** [comment]
Committed your suggestion, thanks!

**Review by smbuthia — APPROVED** [review]
Looks good to go.

**Pull request merged**